lib/trento/clusters/cluster.ex

```diff
@@ -173,7 +174,8 @@ defmodule Trento.Clusters.Cluster do
       },
       %HostAddedToCluster{
         cluster_id: cluster_id,
-        host_id: host_id
+        host_id: host_id,
+        cluster_status: :online
```

Comment:
Can we add a new enum in `trento/clusters/enums/online_status.ex` as `Trento.Clusters.Enums.OnlineStatus`.
So we can use the enum instead of putthing the atom in every plase.